### ISSUE TITLE
`map-markers-overlay` with continuous vars, binSpec type "numBins" is behaving like "binWidth"

### ISSUE THREAD

**Issue by bobular** [VEuPathDB/EdaDataService#196]
If the request contains `binSpec: { type: 'numBins', value: 8 }` the service is interpreting this as binWidth of 8
:point_down: 

![image](https://user-images.githubusercontent.com/308639/173691344-f30cb360-b549-42de-8d57-fccd689d22ff.png)

**Comment by bobular** [comment]
Moved to b60 because it could unearth back end issues.
Currently map viz client is calculating bin width from `(max-min)/8`.

**Comment by bobular** [comment]
Hi @d-callan 
I think there's some confusion about the meaning of `binSpec.type` on client and server side.

On the client side I had assumed that `binSpec: { type: 'numBins', value: 10 }` would mean "give me 10 bins". But that's not correct.

On the server side `binSpec.type` is handled as `binReportValue` and is concerned, as the name suggests, with the response.

The map-markers-overlay code in this repo and plot.data doesn't seem to allow a certain number of bins to be requested.  The client is currently happy to calculate the binWidth.  Should we change the scope of this ticket to implement "requested number of bins" (as very low priority) or should we just close it?

From a client perspective, I can't imagine offering users a "number of bins" control for this (or other) visualizations.  For the map overlay, I could possibly imagine a binWidth control but not in the next six months!

**Comment by d-callan** [comment]
You're right, that's exactly what num bins is meant to mean. Plot.data was never aware of the concept, and only accepts bin widths. but the data service is meant to take num bins and find a bin width if needed before asking plot.data for anything. I can look into why that's not happening.

**Comment by bobular** [comment]
Ah, I see. Yeah, the data service is not currently interrogating the value of `binSpec.type`. I don't think it's on anyone's urgent wish-list to have num-bins controls?

**Comment by bobular** [comment]
> I'd also be ok removing support for num bins officially, just not sure how others feel about asking users for num bins vs bin width etc

@d-callan 
I guess we could decide on this at the next data viz or maybe an EDA UX meeting?

Can definitely remove from b60 must-haves...

**Comment by d-callan** [comment]
Yea. I think that's probably a ux topic (@danicahelb)
